pkg/util/test/util.go

```diff
@@ -50,6 +51,35 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
+const (
+	// SpecDir is a directory with all the specs
+	SpecDir = "./operator-test"
+
+	// DefaultPxNamespace is a default namespace for StorageCluster
+	DefaultPxNamespace = "kube-system"
+
+	// PxReleaseManifestURLEnvVarName is a release manifest URL Env variable name
+	PxReleaseManifestURLEnvVarName = "PX_RELEASE_MANIFEST_URL"
+
+	// PxRegistryUserEnvVarName is a Docker username Env variable name
+	PxRegistryUserEnvVarName = "REGISTRY_USER"
+	// PxRegistryPasswordEnvVarName is a Docker password Env variable name
+	PxRegistryPasswordEnvVarName = "REGISTRY_PASS"
+
+	// DefaultValidateDeployTimeout is a default timeout for deployment validation
+	DefaultValidateDeployTimeout = 900 * time.Second
+	// DefaultValidateDeployRetryInterval is a default retry interval for deployment validation
+	DefaultValidateDeployRetryInterval = 30 * time.Second
+	// DefaultValidateUpgradeTimeout is a default timeout for upgrade validation
+	DefaultValidateUpgradeTimeout = 1400 * time.Second
+	// DefaultValidateUpgradeRetryInterval is a default retry interval for upgrade validation
+	DefaultValidateUpgradeRetryInterval = 60 * time.Second
+	// DefaultValidateUninstallTimeout is a default timeout for uninstall validation
+	DefaultValidateUninstallTimeout = 900 * time.Second
+	// DefaultValidateUninstallRetryInterval is a default retry interval for uninstall validation
+	DefaultValidateUninstallRetryInterval = 30 * time.Second
```

Comment:
Why not leave these constants in the integration_test package? Isn't it better to have them in the caller package?

Reply:
Moved back to common.go